content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,730 @@
+bottom right corner. To send the email, click :guilabel:`Send` and the email will be sent to the
+applicant. The email then appears in the chatter.
+
+Stage flow
+==========
+
+In order to move an applicant's card from one stage to another, the applicant's card can either be
+dragged and dropped to the desired stage, or the stage can be modified on the applicant's card.
+
+To change the stage on the applicant's card, click on the applicant's card in the kanban view to go
+to a detailed view of the card. The current stage for the card is highlighted in purple at the top,
+above the card. Simply click the desired stage for the card, and the stage changes. A note
+indicating the stage change appears in the chatter. Navigate back to the kanban view by clicking on
+:guilabel:`Applications` in the breadcrumb menu in the top left, and the applicant's card now
+appears in the selected stage.
+
+.. image:: recruitment/stage-change.png
+   :align: center
+   :alt: Change the stage of an applicant by clicking on the desired stage at the top of the
+         applicant's card.
+
+.. note::
+   The following information is based on Odoo's default recruitment pipeline flow. Be advised that
+   if modifications are made to create a customized recruitment pipeline flow, it will vary from the
+   following information.
+
+Initial qualification
+---------------------
+
+All applicants appear in the :guilabel:`Initial Qualification` stage after applying for a job
+position online. When the applicant's card is created, Odoo automatically populates the applicant's
+:guilabel:`Name`, :guilabel:`Email address`, and :guilabel:`Phone number` on the applicant's card.
+This information is required when applying for a job position.
+
+If the applicant entered any information in the :guilabel:`Short Introduction` section of the online
+application, it is populated in the :guilabel:`Application Summary` tab at the bottom of the
+applicant's card. If a resumé was attached to the online application, the resumé appears in the
+chatter, and is also stored in the *Documents* application. The resumé is accessible through both
+the *Recruitment* and *Documents* applications.
+
+Send interview
+~~~~~~~~~~~~~~
+
+At any point in the hiring process, an *interview* can be sent to the applicant to obtain more
+information. These interviews are custom-made, and can be formatted in a variety of ways.
+
+Odoo uses the term *interview*, but these can be thought of as questionnaires, surveys, tests,
+certifications, etc. Custom interviews can be formatted to suit each individual job position's
+needs. For example, a job position for a computer programmer could have an interview in the form of
+a programming quiz to determine the skill level of the applicant. A job position for a restaurant
+server could have a questionnaire inquiring about the applicant's availability, if the desired
+applicant needs to be available on weekend evenings. For more information on creating and editing
```

Comment:
Yes- this is the Job Positions document I was mentioning earlier. In this PR I have a note in here (near the top) stating that this document must be published after the job positions one. Currently, it does not exist to link to which is why it does not render, but once it is published, it will (and I will test it again, to make sure! =) 

This was my solution to the issue I had and you found in my fleet docs- where things were doubled up because you were asked to do the same thing in a different section, etc.